staff-commands/list-temproles.md

```diff
@@ -21,13 +21,9 @@ None
 ### **Usage**
 
 ```
-m:list-temproles (all|user|role) <extras>
+/list-temproles (all|user|role) <extras>
```

Comment:
Command usage changed (takes an argument which is either the role or the user, if none then it behaves like the old `all`)